assets/training/model_management/environments/foundation-model-inference/context/Dockerfile

```diff
@@ -1,22 +1,24 @@
-# Use the base image that includes the necessary dependencies
-FROM ghcr.io/huggingface/text-generation-inference:0.9
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch200:{{latest-image-tag}}
```

Comment:
We'll need to update this. Per @savita, the new namespace for acpt images are in the following format: 
`FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py38-torch1131:{{latest-image-tag:biweekly\.\d{6}\.\d{1}}}`

Reply:
yes, you will need to replace with below base image for 2.0.1 with updated namespace
FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py38-torch201:{{latest-image-tag:biweekly\.\d{6}\.\d{1}}}